satellite/rewards/display.go

```diff
@@ -78,9 +80,18 @@ func (offers Offers) OrganizeOffersByType() OfferSet {
 func createPartnerSet() PartnerSet {
 	partners := LoadPartnerInfos()
 	var ps PartnerSet
-	for _, partner := range partners {
+	keys := make([]string, len(partners))
+	i := 0
+	for k := range partners {
```

Comment:
I think this would be easier to read as: `for i := 0; i < len(partners); i++{ keys[i] = partners[i] }`

Reply:
`partners` is a map so I can't use index as the key to access the value